.github/workflows/build-container.yml

```diff
@@ -1,32 +1,132 @@
 name: "CI"
 on:
   push:
-    branches:
-      - devel
   pull_request:
-    branches:
-      - devel
   schedule:
     - cron: '0 0 * * *'
   workflow_dispatch:
-env:
-  HUGEPAGES: 256
 
 jobs:
-  build-and-test:
+  build:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
         with:
+          # git submodule update --init --recursive
           submodules: recursive
 
       - name: Build container images
-        run: make build
+        run: |
+          make build
+
+      - name: Save container images
+        run: |
+          . .env
+          docker save quay.io/ceph/nvmeof:$NVMEOF_VERSION > nvmeof.tar
```

Comment:
I'm not familiar with all the container registries, but at least quay.io doesn't automatically set the `latest` tag, so you need to decide what's chronologically "latest" (`docker tag quay.io./ceph/nvmeof:latest`) and push that to the registry. It'd be nice to rely on some registry automation to do so.

Personally, I would stay with explicit version pinning, since `latest` could result in non-deterministic/non-reproducible deployments. If we want to avoid having to manually increase micro/bugfix versions (`x.y.Z`), we could use a tag like `v1.0` instead, and refresh that after every bugfix... What do you think?

Reply:
sounds good to me, we can leave it as it is for now.